### PR TITLE
corrects formatting as per clang-format

### PR DESCRIPTION
**Pull request by d2r**
Milestone: 8.0.0
Labels: Cleanup

### CODE REVIEW

**Comment by shinrich** [comment]
Weren't these issues caught by previous clang-format checks?

**Comment by d2r** [comment]
> Weren't these issues caught by previous clang-format checks?

Apparently not... I found this surreptitiously after running `make clang-format` in my working tree.

I am not sure how it happened, or why it remains undetected in PRs.

**Comment by d2r** [comment]
@zwoop This is an open question: Do we know why our automated builds do not catch this file?

**Comment by bryancall** [comment]
@d2r I am not seeing this when I run clang-format on macOS

**Comment by d2r** [comment]
> @d2r I am not seeing this when I run clang-format on macOS

I saw this in a RHEL6 environment. I have not tried others yet.

File: cmd/traffic_ctl/traffic_ctl.cc [hunk 1]
```diff
@@ -30,13 +30,27 @@
 
 AppVersionInfo CtrlVersionInfo;
 
-const char *CtrlMgmtRecord::name() const { return this->ele->rec_name; }
+const char *
```
Comment (bryancall) [inline cmd/traffic_ctl/traffic_ctl.cc]: This is what I see when I look at the current master:
``
const char *
CtrlMgmtRecord::name() const
{
  return this->ele->rec_name;
}

Comment (d2r) [reply cmd/traffic_ctl/traffic_ctl.cc]: This was recently fixed... checking...

Comment (d2r) [reply cmd/traffic_ctl/traffic_ctl.cc]: This is a duplicate of #3271 I'll close it.

**Comment by d2r** [comment]
This is a duplicate of #3271

**Pull request closed without merging**